llvm/include/llvm/IR/VectorBuilder.h

```diff
@@ -87,6 +87,9 @@ class VectorBuilder {
     StaticVectorLength = ElementCount::getFixed(NewFixedVL);
     return *this;
   }
+
+  FastMathFlags &getFastMathFlags() const { return Builder.getFastMathFlags(); }
```

Comment:
Might be good to return `const FastMathFlags` and document

Reply:
Done, thanks!